antenna-documentation/src/site/markdown/analyzers/csv-analyzer-step.md.vm

```diff
@@ -0,0 +1,113 @@
+* `Version`: Version of the release.
+* `Coordinate Type`
+
+#[[###]]# Optional Attributes
+
+The following attributes are optional and can be added to enrich the information in the individual artifacts
+
+* `Effective License`: The licenses effective during the ${docName} run in the SPDX license expression that displays the relationship of the licenses as one String.
+* `Declared License`: The licenses declared for the component in the SPDX license expression that displays the relationship of the licenses as one String.
+* `Observed License`: The licenses observed during the ${docName} run in the SPDX license expression that displays the relationship of the licenses as one String.
+* `Copyrights`: Copyrights of the release. Multiple copyrights are to be separated  by either '\n' or '+'.
+* `Hash`: A hash connected to the variable. To see how to insert multiple hashes see the section "Multiple Hashes for one Artifact"
+* `Source URL`: Link to the source archive. E.g., used to aggregate the sources of the used OSS components for the FOSS compliance bundle.
+* `Repository Tag UR`L: This is a link to the OSS repos tag of the release used.
+* `Software Heritage ID`: A release ID in software heritage in the format "swh:1:rel:*".
+* `Clearing State`: Data Approval State.
+* `Change Status`: OSS Usage.
+* `CPE`: Common Platform Enumeration ID.
+* `Filename`: Name of the file of the dependency.
+
+#[[###]]# Change of format
+
+Within the old CSV format there were attributes (= columns) that were required:
+
+* `Artifact Id`: still exists
+* `Group Id`: still exists
+* `Version`: still exists
+* `License Short Name`: was changed to `Effective License`
+* `License Long Name`: was removed since it is not conforming with the SPDX expression of the licenses
+* `Filename`: still exists, but is not required anymore
+
+#[[###]]# Multiple Hashes for one Artifact
+If you have an artifact you associate multiple hashes with, this can be done in the following way:
+
+```csv
+"Artifact Id","Group Id","Version","Coordinate Type","Hash"
+commons-csv,org.apache.commons,1.4,mvn,620580a88953cbcf4528459e485054e7c27c0889
+commons-csv,org.apache.commons,1.4,mvn,b0060ed8397bfec39b397807f63e778618f324ce
+```
+
+You can simply insert the new hash in a new row with the same artifact coordinates.
+This will then get merged with the already existing artifact.
+Artifacts get identified by their coordinates.
+So, if you do not give the coordinates and just an empty row with hashes, the content will not get matched to its corresponding artifact.
+
+#[[###]]# Multiple Copyrights Example
+Here we provide an example on how to have multiple copyrights in one entry.
+There are three copyrights in total, each separated by "/n", the copyright separator.
+
+```csv
+"Artifact Id","Group Id","Version","Coordinate Type","Copyrights"
+commons-csv,org.apache.commons,1.4,mvn,"Copyright 2005-2016 The Apache Software Foundation/nCopyright 2020 Fake Company/nCopyright 2020 Fake the 2nd"
```

Comment:
Really /n as line separator?

Reply:
Will open a follow up issue as just discussed. 
Since '/n' is also the delimiter used in the `CopyrightStatement` class, we need to check if this will be an issue anywhere in the places that handle copyright statements.